modules/globebrowsing/videoslide.asset

```diff
@@ -0,0 +1,27 @@
+local slide = {
+  Identifier = "TestVideoSlide2",
+  File = "C:/Users/ylvaselling/Downloads/equirectangular-projection-testmovie02.mp4",
```

Comment:
Same here. Don't use a local file

This test asset should not be included, unless we put a test video on the sync server